Content.Client/Preferences/UI/HumanoidProfileEditor.xaml.cs

```diff
@@ -434,13 +428,78 @@ public partial class HumanoidProfileEditor : Control
 
             #endregion Left
 
-            #region Right
+            if (preferencesManager.ServerDataLoaded)
+            {
+                LoadServerData();
+            }
+
+            preferencesManager.OnServerDataLoaded += LoadServerData;
+
+            IsDirty = false;
+        }
+
+        private void OnSkinColorOnValueChanged(Range range)
+        {
+            if (Profile is null) return;
+
+            var skin = _prototypeManager.Index<SpeciesPrototype>(Profile.Species).SkinColoration;
+
+            switch (skin)
+            {
+                case SpeciesSkinColor.HumanToned:
+                {
+                    var rangeOffset = (int) range.Value - 20;
 
-            #region Preview
+                    float hue = 25;
+                    float sat = 20;
+                    float val = 100;
+
+                    switch (rangeOffset)
```

Comment:
This would do the same thing as the switch
if (rangeOffset <= 0)
{
	hue += Math.Abs(rangeOffset);
}
else
{
	sat += rangeOffset;
	val -= rangeOffset;
}

Reply:
Not my code, and the switch feels clearer here imo.

Reply:
Welp mirror says to change it so I did.